test/spec/browser_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'yaml'
+require_relative './app_spec.rb'
+
+config = YAML.load_file('config.yml')
+webdriver_url = "http://#{config['sauce_username']}:#{config['sauce_access_key']}@ondemand.saucelabs.com:80/wd/hub"
+
+browsers = [
+  :chrome,
```

Comment:
It's not a key name, just an entry in the array :) Sauce (Selenium?) expands it to {browserName: :chrome}.

An improvement(...hopefully) I'm working on now should be more clear – instead of an array, a hash that maps slugs (of our invention) to Selenium browser capability hash, like:

```
browser_definitions = {
  safari_ipad_ios10: {
    browserName: 'Safari',
    deviceName: 'iPad Simulator',
    platformName: 'iOS',
    platformVersion: '10.0',
    javascriptEnabled: true,
    appiumVersion: '1.6.3'
  },

  chrome54_macos1012: {
    browserName: 'chrome',
    version: '54.0',
    platform: 'macOS 10.12'
  },

  chrome54_windows10: {
    browserName: 'chrome',
    version: '54.0',
    platform: 'Windows 10'
  },

  chrome48_linux: {
    browserName: 'chrome',
    version: '48.0',
    platform: 'Linux'
  }
}
```

You could then run the spec against a subset of browsers similar to `run :chrome48_linux` and something would look up what `:chrome48_linux` means in Selenium terms as needed. Vaguely similar to what they're doing in this example repo except in rakefile form: 

https://github.com/saucelabs-sample-test-frameworks/Ruby-RSpec-Selenium/blob/master/Rakefile

(The issue with their approach is all the tests get run but with no ability to track what happens to them...)